content/docs/2.10/concepts/admission-webhooks.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Admission Webhook"
+description = "Automatically validate resource changes to prevent misconfiguration and enforce best practices"
+weight = 100
++++
+
+## Admission Webhooks
+
+> 💡 The Admission Webhooks are an opt-in feature and will become an opt-out feature as of KEDA v2.12.
+
+Admission webhooks require a valid certificate to expose the server for the api-server. To achieve this KEDA generates its own certificates and stores them in a secret.
+
+While this is a good starting point, some end-users may want to use their own certificates which are generated from their own CA in order to improve security. This can be achieved by disabling the certificate rotation via setting the console argument `enable-cert-rotation=false` (or removing it) and mounting the `tls.crt` and `tls.key` files to `/certs`. The path is also configurable using the console argument `webhooks-cert-dir=PATH-FOR-CERTS`.
+
+By default, the admission webhooks are registered with `failurePolicy: Ignore`, this won't block the resources creation/update when the admission controller is not available. To ensure that the validation is always required and perform validation, setting `failurePolicy` to `Fail` is required.
+
+### Prevention Rules
+
+KEDA will block all incoming changes to `ScaledObject` that don't match these rules:
+
+- The scaled workload (`scaledobject.spec.scaleTargetRef`) is already autoscaled by another other sources (other ScaledObject or HPA).
+- CPU and/or Memory trigger are used and the scaled workload doesn't have the requests defined. **This rule doesn't apply to all the workload types, only to `Deployment` and `StatefulSet`.**
```

Comment:
This content is good, but in my opinion this is a mix of two types of docs:
- **Concepts** - What is it doing and how does it help me?
- **Configuration/Ops** - How do I manage certs.

I believe we should move the certs to a new "operate" page for "admission webhooks" or extend it to our existing "cluster" doc (although I think that one is not the best place and will burry the info.

I personally think all we need to have in this doc is:

```suggestion

The Admission Webhooks are used to automatically validate resource changes to prevent misconfiguration and enforce best practices.

This is achieved by intercepting and validating all resource changes based on a set of [prevention rules](#prevention-rules).

<<Visual with interaction between control plane, webhooks and eventually operator processing CRD if applicable>>

### Prevention Rules

KEDA will block all incoming changes to `ScaledObject` that don't match these rules:

- The scaled workload (`scaledobject.spec.scaleTargetRef`) is already autoscaled by another other sources (other ScaledObject or HPA).
- CPU and/or Memory trigger are used and the scaled workload doesn't have the requests defined. **This rule doesn't apply to all the workload types, only to `Deployment` and `StatefulSet`.**

Reply:
I have split into 2 different pages, but I feel both like too short :( 
PTAL @tomkerkhove

Reply:
LGTM